test/tests.js

```diff
@@ -1107,6 +1117,106 @@ describe('Braze Forwarder', function() {
         (window.braze.getUser().yearOfBirth === null).should.equal(true);
     });
 
+    it('decodeSubscriptionGroupMappings should return parsed subscriptionGroupIds map when proper setting is given', function () {
+        // sample subscriptionGroupMapping from config
+        var subscriptionGroupMapping = '[{&quot;jsmap&quot;:null,&quot;map&quot;:&quot;subscriptionGroupTest1&quot;,&quot;maptype&quot;:&quot;UserAttributeClass.Name&quot;,&quot;value&quot;:&quot;00000000-0000-0000-0000-000000000000&quot;},{&quot;jsmap&quot;:null,&quot;map&quot;:&quot;subscriptionGroupTest2&quot;,&quot;maptype&quot;:&quot;UserAttributeClass.Name&quot;,&quot;value&quot;:&quot;00000000-0000-0000-0000-000000000001&quot;}]';
+
+        // initialize Braze kit with subscriptionGroupMappings
+        mParticle.forwarder.init(
+            {
+                apiKey: '123456',
+                subscriptionGroupMapping: subscriptionGroupMapping,
+            },
+            reportService.cb,
+            true,
+            null
+        );
```

Comment:
pretty sure this can be removed since you are only testing the method and you don't need to do a full initialization fo the forwarder to do so
```suggestion
```

Reply:
@rmi22186 - just removed it, thanks for the feedback!